components/Map.js

```diff
@@ -51,6 +65,14 @@ export default function Map() {
           );
         })}
       </StyledMapContainer>
+      <StyledPositionBtn onClick={handleSetView}>
+        <Image
+          src={locateMeButton}
+          alt="locateMeButton"
```

Comment:
try to give real hints, what the button does. Dont use Buttum in the alt text. 
"locate me" or "locate my position" would work here.

Reply:
i changed it :)